app/views/docs/main.phtml

```diff
@@ -57,15 +57,20 @@ $getSpecsUrl = function($type, $platform) use ($latestVersion) {
 
 <div class="zone xxl">
 
-    <p class="margin-bottom">Welcome to our documentation section. Here you will be able to find quick guides to setup and integrate with Appwrite on your platform of choice. You can also review full and detailed API references to all of our services, tools, and features.</p>
+    <p class="margin-bottom">This documentation contains all the information you need to start building Web, Flutter, Android, or iOS apps with Appwrite. You can also review the complete and detailed API references to our services, tools, and features.</p>
     
+    <div class="notice">
+        <h2>Can't wait to get started?</h2>
+        <p><a href="https://cloud.appwrite.io/">Create your first project now</a> and start building on Appwrite Cloud.</p>
+    </div>
+
     <h2 class="margin-bottom">Overview</h2>
 
-    <p>Appwrite is an end-to-end backend server that aims to abstract the complexity of common, complex, and repetitive tasks required for building a modern app.</p>
+    <p>Appwrite is a secure, open-source backend that abstracts the complexity of common, complex, and repetitive tasks required for building a modern app.</p>
 
-    <p>Appwrite provides you with a set of APIs, tools, and a management console UI to help you build your apps a lot faster, and in a much more secure way. Within Appwrite's different services, you can find user authentication and account management, user preferences, database and storage persistence, cloud functions, localization, image manipulation, and more.</p>
+    <p>Appwrite provides you with a set of APIs, tools, and a management console UI to help you build your apps quickly without compromising security. Appwrite includes features like user authentication and account management, user preferences, database and file storage, cloud functions, localization, image manipulation, and more.</p>
```

Comment:
Can we use some other word other than security ? This former part of the sentence does not showcase how security could be compromised

Reply:
No, it's that you can build **quickly** and still not compromise on **security**. One less thing to worry about. I think security is the only word here as well.